build.sh

```diff
@@ -170,60 +164,22 @@ function docker_build_with_version {
 
   pull_image "$dockerfile"
 
+  is_podman() {
+    local _bin
+    _bin=$(docker --version | cut -d" " -f1)
+    [[ $_bin == "podman" ]] && echo true
```

Comment:
Why do we echo true here?

Reply:
You are right, I will just return 1 if not podman.